examples/SingleDatePickerWrapper.jsx

```diff
@@ -15,6 +15,7 @@ const propTypes = {
   // example props for the demo
   autoFocus: PropTypes.bool,
   initialDate: momentPropTypes.momentObj,
+  isRTL: PropTypes.bool,
```

Comment:
Same. This should be a part of SingleDatePickerShape so you don't need to add it explicitly to the wrapper proptypes